app/ide-desktop/lib/content/src/style.css

```diff
@@ -73,11 +73,6 @@
 
 /* End of fonts */
 
-html,
-body {
-  height: 100vh;
-}
```

Comment:
Why this change? On some browsers you need to set it in order for the WebGL canvas to occupy 100% of the view. Again, such changes should be explained in the PR (e.g. by adding comments from the author of the PR) to justify why that particular change was necessary.

Reply:
i think it was to fix dashboard styles? or i just removed it while debugging something. either way can't see any issues when adding this back in so i'll just add it back in